gnovm/tests/files/closure11.gno

```diff
@@ -0,0 +1,22 @@
+package main
+
+type fooer interface{}
+
+var f fooer = 1
+
+func trymake() func() {
+	switch v := f.(type) {
+	case int:
+		return func() {
+			println(v)
+		}
+	default:
+	}
+	return func() {}
+}
+func main() {
+	trymake()()
+}
+
```

Comment:
Do we want to add a golden test for `// Preprocessed:`?

Reply:
added.